### PR TITLE
replace reference to `TEAMCITY_BUILD` with `GITHUB_RUN_NUMBER`

### PR DESCRIPTION
**Pull request by michaelbjacobson**
This PR is a follow-up from the Teamcity to Github Actions migration. It removes the final reference to Teamcity from the repo.

### CODE REVIEW

File: webpack.common.js [hunk 1]
```diff
@@ -1,20 +1,21 @@
...
 	path: path.resolve(__dirname, './dist/'),
 });
 
 const definePlugin = new webpack.DefinePlugin({
-	WEBPACK_BUILD: process.env.TEAMCITY_BUILD
-		? `'${process.env.TEAMCITY_BUILD}'`
+	WEBPACK_BUILD: process.env.GITHUB_RUN_NUMBER
+		? `'${process.env.GITHUB_RUN_NUMBER}'`
 		: `'DEV_${new Date().getTime()}'`,
 	GIT_COMMIT_HASH: process.env.BUILD_VCS_NUMBER
```
Comment (andrewHEguardian) [inline webpack.common.js:20]: what is `BUILD_VCS_NUMBER`? is that also a teamcity thing

Comment (jamesmockett) [reply webpack.common.js:20]: Yeah, that's another TeamCity environment variable, although it has a fallback. The commit hash is used by prout to determine when a change is live: https://manage.theguardian.com/_prout

[`GITHUB_SHA`](https://docs.github.com/en/actions/learn-github-actions/variables) looks like a suitable replacement?

Comment (michaelbjacobson) [reply webpack.common.js:20]: Good catch, I just assumed it was something else entirely. `GITHUB_SHA` looks perfect!

**Review by andrewHEguardian — APPROVED** [review]
Looks good to me 👍

**Pull request merged**